docs/platform/concepts/maintenance-window.rst

```diff
@@ -24,4 +24,12 @@ In case of **Apache Kafka®** and **OpenSearch®** the service DNS address resol
 Maintenance updates
 ~~~~~~~~~~~~~~~~~~~
 
-Security updates, platform updates that affect reliability or stability of the service nodes, and quarterly patch updates are always mandatory. Other updates are initially optional. Advance notice is given for all updates. After optional updates have been available for six months, they become mandatory and are applied on the next week's maintenance window at the earliest. This means you have at least 7 days advance notice with exception of critical security updates. These critical updates are applied in the maintenance window of the current week. During service upgrades, maintenance updates are automatically applied and do not require any action from you. 
\ No newline at end of file
+Security updates, platform updates that affect reliability or stability of the service nodes, and quarterly patch updates are always mandatory. Other updates are initially optional. Advance notice is given for all updates. After optional updates have been available for six months, they become mandatory and are applied on the next week's maintenance window at the earliest. This means you have at least 7 days advance notice with exception of critical security updates. These critical updates are applied in the maintenance window of the current week. During service upgrades, maintenance updates are automatically applied and do not require any action from you.
+
+Pending maintenance updates can be viewed in three places. 
```

Comment:
Please use the active voice (vs the passive voice) whenever possible:
You can view maintenance updates pending for your service using the following:

Reply:
Doneso. :)